examples/intervention_mode/README.md

```diff
@@ -0,0 +1,253 @@
+# Intervention Mode Demo
+
+## Introduction
+
+This is a prototype of an Inspect agent with human intervention. It utilizes Inspect's [Interactivity features](https://inspect.ai-safety-institute.org.uk/interactivity.html). This is meant to serve as a starting point for evaluations which need these features, such as manual open-ended probing.
+
+This gives an overview of this setup and how to customize it.
+
+## Installation
```

Comment:
We indeed don't need rich (it will be there if Inspect is there) so I don't think you need anything about Python packages in the Installation section (you can probably nix the whole section)

Reply:
Removed section